buildflow/runtime/runner.py

```diff
@@ -94,20 +92,16 @@ class Runtime:
     def __init__(self):
         self._processors: Dict[str, _ProcessorRef] = {}
         self._session = _load_session()
-        parser = argparse.ArgumentParser()
-        parser.add_argument('--disable_usage_stats',
-                            action='store_true',
-                            default=False)
-        args, _ = parser.parse_known_args(sys.argv)
-        self._enable_usage = True
-        if args.disable_usage_stats:
-            self._enable_usage = False
-
-    def run(self, streaming_options: options.StreamingOptions):
-        if self._enable_usage:
+
+    def run(self, streaming_options: options.StreamingOptions,
```

Comment:
nit: Cant remember if this is the top level run() method or not, but if so, consider adding (*, ...) so users have to pass keyword args

Reply:
done.